core/controllers/base.py

```diff
@@ -510,6 +522,19 @@ def handle_exception(self, exception, unused_debug_mode):
         UserFacingExceptions.TemporaryMaintenanceException)
 
 
+class SeedFirebaseHandler(webapp2.RequestHandler):
```

Comment:
Yep we'll rip this out eventually, so I don't think it's worth adding logic. My hope is that this will become much simpler once we don't need to worry about backups taking us to a state without any firebase association models.